### ISSUE TITLE
ToggleButtonTheme?

### ISSUE THREAD

**Issue by jgarza9788** [flutter/flutter#42095]
not sure the correct way to set the ToggleButtonTheme.

but here is a piece of my code 
```dart
ThemeData purpleRain() {

  return ThemeData.light().copyWith(
    primaryColor: Colors.deepPurple,
    accentColor: Colors.deepPurple,
    canvasColor: Colors.deepPurple[200],
    backgroundColor: Colors.deepPurple,
    splashColor: Colors.deepPurple[900],
    scaffoldBackgroundColor: Colors.deepPurple[200],
    toggleableActiveColor: Colors.deepPurple,
/*Debug !...these lines generate an error*/
//    toggleButtonsTheme: ToggleButtonsTheme(
//      data: ToggleButtonsThemeData(
//        color: Colors.deepPurple,
//      ),
//    ),
    buttonTheme: ButtonThemeData(
      buttonColor: Colors.deepPurple[500],
      splashColor: Colors.deepPurpleAccent[100],
      shape: RoundedRectangleBorder(
        borderRadius: BorderRadius.circular(50.0),
      ),
      colorScheme: ColorScheme(
        primary: Colors.deepPurple[900],
        primaryVariant:Colors.deepPurple,
        secondary: Colors.deepPurple[100],
        secondaryVariant: Colors.deepPurple,
        surface: Colors.deepPurple,
        background: Colors.deepPurple,
        error: Colors.deepPurple,
        onPrimary: Colors.deepPurple,
        onSecondary: Colors.deepPurple,
        onSurface: Colors.deepPurple,
        onBackground: Colors.deepPurple,
        onError: Colors.deepPurple,
        brightness: Brightness.light,
      ),
      textTheme: ButtonTextTheme.primary,
    ),
[...]
```

**Comment by janmoppel** [comment]
Hello @jgarza9788 !

Thank you for posting.
Can you please provide more details, such as the error itself and the output of the `flutter doctor -v` command output? 

Your question seems to be a good fit for the [StackOverflow](https://stackoverflow.com/tags/flutter) as well. Flutter has a very active and helpful community there, so feel free to post it there too.

Also, see other support options here: https://flutter.dev/community

**Comment by hanneskuettner** [comment]
Seems like this is a duplicate of #40992 and fixed in #40994. The fix is available in v1.10.7.

**Comment by HansMuller** [comment]
CC @shihaohong 

https://github.com/flutter/flutter/issues/42095#issuecomment-538905497 is correct, sorry about that. I'm going to close this issue.
